CHANGELOG.md

```diff
@@ -9,7 +9,7 @@ This file is used to list changes made in each version of the aws-parallelcluste
 **CHANGES**
 
 - Drop support for SGE and Torque schedulers.
-- Change tags prefix from `aws-parallelcluster-` to `parallelcluster:`.
+- Use tags prefix `parallelcluster:`.
```

Comment:
I'd call out we're removing Application tag, renaming ClusterName to parallelcluster:cluster-name etc because users might rely on them.